src/classes/metadata/slides_notes.vala

```diff
@@ -1,11 +1,11 @@
 /**
- * Storage for the notes of a presentation
+ * Storage for the jump points
```

Comment:
Why did you changed the comments and removed the copyright notice but didn't changed anything?